### PR TITLE
chore(tact-picker): ensures cursor looks like a text on whole control

### PR DESCRIPTION
**Pull request by bsunderhus**
## Previous Behavior

image

## New Behavior

1. ensures cursor looks like a text on whole control, even on dead parts of it (everything bellow secondary action and expand icon is not used as input)
2. adds exception on cypress for false positive error about ResizeObserver (https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded , https://github.com/cypress-io/cypress/issues/20341)

image

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-tag-picker-preview</samp> <br /> <abbr title='bundle-size/Package.fixture.js'>@fluentui/react-tag-picker-preview - package</abbr> | `185.641 kB`<br />`54.844 kB` | `186.223 kB`<br />`55.005 kB` | `582 B` <br />`161 B` |

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ff79ac88545c9cbc7966f75f61a9d506f3ebb963'>ff79ac88545c9cbc7966f75f61a9d506f3ebb963</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx [hunk 1]
```diff
@@ -13,6 +13,18 @@ import { TagPickerOption } from '../TagPickerOption/TagPickerOption';
 import { Avatar } from '@fluentui/react-avatar';
 import { Button } from '@fluentui/react-button';
 
+/**
+ * This error means that ResizeObserver
+ * was not able to deliver all observations within a single animation frame.
+ * https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded
+ */
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
+Cypress.on('uncaught:exception', err => {
```
Comment (ling1726) [inline packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx:22]: why is this necessary?

Comment (bsunderhus) [reply packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx:22]: Cypress is failing due to some ResizeObserver loop error that can't be reproduced outside of cypress?!

Comment (bsunderhus) [reply packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx:22]: https://github.com/cypress-io/cypress/issues/20341

Comment (ling1726) [reply packages/react-components/react-tag-picker-preview/src/components/TagPicker/TagPicker.cy.tsx:22]: can you link that issue in the comment too?

**Review by ling1726 — APPROVED** [review]
